### DISCUSSION TITLE
Is it possible to add a script in the dialogue?

### DISCUSSION THREAD

**Discussion by brojo265** [nathanhoad/godot_dialogue_manager#206] [Q&A]
Im currently making a game and I was wondering if it was possible to add a script to the dialogue?
For example, I want to add a script to my game where after the specific dialogue has ended it would switch scenes to another scene.

Although when I tried to do it with, get_tree().change_scene("res://level_2.tscn"), the in-game dialogue just typed out the code instead of actually executing it.  
![image](https://github.com/nathanhoad/godot_dialogue_manager/assets/37369580/32d8470d-a57d-4467-83d6-50bf69c8c8ae)
![image](https://github.com/nathanhoad/godot_dialogue_manager/assets/37369580/ade2d15e-3b71-46b4-9fe1-26105b7a31ce)

If it isn't possible then is there an alternative way of doing this?

**Comment by nathanhoad** [comment]
To change scenes you'd have to write a [mutation function](https://github.com/nathanhoad/godot_dialogue_manager/blob/main/docs/Writing_Dialogue.md#mutations) that changes the scene and then call it from the dialogue with a `do` line.
